tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -729,6 +717,7 @@ def _test_cpu_cpuid_snapshot(context):
         / context.kernel.base_name()
         / cpu_template_dir
     )
+    shutil.rmtree(snapshot_artifacts_dir, ignore_errors=True)
```

Comment:
This can go in a separate commit as it is an independent fix, as I understand?
We can then use conventional tags in those commits: `fix(tests): ` and `feature(tests): `.
Also, can be more specific in the commit message on which templates are being tested.